tests/integration/telemetry/stats/prometheus/extsionproviders/accesslogs_test.go

```diff
@@ -0,0 +1,62 @@
+//go:build integ
+// +build integ
+
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package extsionproviders
+
+import (
+	"testing"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/istio"
+	"istio.io/istio/pkg/test/framework/label"
+	"istio.io/istio/pkg/test/framework/resource"
+	common "istio.io/istio/tests/integration/telemetry/stats/prometheus"
+)
+
+func TestMain(m *testing.M) {
```

Comment:
Can we just add the access log mesh config to the existing test suite and avoid adding new suite https://github.com/istio/istio/blob/a3a07b85065a1ebd6eef2ba7e633eb311a628b60/tests/integration/telemetry/stats/prometheus/nullvm/stats_filter_test.go#L48 or is there some reason we cannot do that?

Reply:
Probably use https://github.com/istio/istio/blob/4a28c571b84adf2eab44d009ef338a2686501510/pkg/test/framework/components/istio/util.go#L180.

Reply:
I'll try to refactor the test case ASAP